static/app/components/search/index.tsx

```diff
@@ -209,7 +210,7 @@ function Search({
   );
 }
 
-const WithRouterSearch = withRouter(Search);
+const WithRouterSearch = Search;
 export {WithRouterSearch as Search, SearchProps};
```

Comment:
```suggestion
export {Search, SearchProps};
```

The extra alias seems redundant now.